GVRf/Framework/framework/src/main/jni/contrib/jassimp2/memory_file.cpp

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
We should probably add a Copyright header. Either the original MIT license or Samsung Apache should be OK, as they are compatible.